.gitignore

```diff
@@ -127,3 +127,4 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+/poetry.lock
```

Comment:
Probably not important for now: It could be helpful to add `poetry.lock` to the repository (`poetry install` will take less time and it improves reproducibility) but every now and then someone needs to update `poetry.lock` (is there a github bot for this?).

Reply:
Hmm i see, probably a good ideia let poetry.lock available. But what do you mean with github bot, I didn't get the ideia.